install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -19,6 +19,14 @@ metadata:
   namespace: openshift-machine-api
 rules:
 
+  - apiGroups:
```

Comment:
does this work for `role` given the infra CRD is cluster scoped? it should be in the machine controllers `ClusterRole`

Reply:
I ran tests, and it passed so probable it does not have any problems

Reply:
not sure which tests you mean, but even if it works can you please fix it and put clusterScope resources under the clusterRole? https://kubernetes.io/docs/reference/access-authn-authz/rbac/ thanks